x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -344,7 +344,7 @@ export async function ensurePreconfiguredPackagesAndPolicies(
         policy!,
         packagePoliciesToAdd!,
         defaultOutput,
-        !created
+        true
```

Comment:
this is the bumpRevision flag, which was previously set to false on create (and true on update), it is used here:
https://github.com/elastic/kibana/blob/b8c1323d5bdc921a6238003bc698bc03750c5320/x-pack/plugins/fleet/server/services/package_policy.ts#L177-L204